logitechmediaserver.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Logitechmediaserver",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/titou386/iocage-plugin-logitechmediaserver.git",
+  "properties": {
+    "boot": "on",
+    "dhcp": "on"
+	  },
+  "pkgs": [
+    "perl5",
+    "p5-libwww",
+    "p5-Font-FreeType",
+    "p5-Data-Dump",
+    "p5-Net-SSLeay",
+    "p5-Crypt-OpenSSL-RSA",
+    "p5-IO-Socket-SSL",
+    "bash",
+    "curl",
+    "flac",
+    "faad2",
+    "lame",
+    "sox"
+    ],
+  "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/quarterly",
```

Comment:
Could this perhaps be changed to `http://pkg.FreeBSD.org/${ABI}/latest` to use the `latest` packagesite and make it more dynamic?

Reply:
> Should be enough to port this PR/plugin/master to the `11.3-RELEASE` branch.

Oohh right, I didn't pay attention to the branches.



> Could this perhaps be changed to `http://pkg.FreeBSD.org/${ABI}/latest` to use the `latest` packagesite and make it more dynamic?

I will modify.